### ISSUE TITLE
Tekken 3D Prime Edition / Hangs

### ISSUE THREAD

**Issue by JohnGodgames** [citra-emu/citra#975]
![tekken](https://cloud.githubusercontent.com/assets/11453507/8817056/d03dc00a-302e-11e5-9be8-674dcf147514.PNG)

When I press start next screen is black and hangs.Where is the problem?

```
[ 268.919648] Service <Error> core\hle\service\service.cpp:Service::Interface::S
yncRequest:68: unknown / unimplemented function 'SuspendDaemons': port=ndm:u, cm
d_buff[1]=0x1
[ 268.921148] Service <Error> core\hle\service\service.cpp:Service::Interface::S
yncRequest:68: unknown / unimplemented function '0x000100C2': port=cecd:u, cmd_b
uff[1]=0x69F00, cmd_buff[2]=0x2, cmd_buff[3]=0x2, cmd_buff[4]=0x20, cmd_buff[5]=
0x14A
[ 268.922148] Service <Error> core\hle\service\service.cpp:Service::Interface::S
yncRequest:68: unknown / unimplemented function '0x00020042': port=cecd:u, cmd_b
uff[1]=0x60, cmd_buff[2]=0x60C, cmd_buff[3]=0xFFFFD68
[ 268.923148] Service <Error> core\hle\service\service.cpp:Service::Interface::S
yncRequest:68: unknown / unimplemented function '0x00070040': port=ndm:u, cmd_bu
ff[1]=0x1
[ 268.924149] Service.CFG <Error> core\hle\service\cfg\cfg.cpp:Service::CFG::Get
ConfigInfoBlock:197: Config block 0xC0000 with flags 2 and size 192 was not foun
d
```

LOG: http://pastie.org/10305270

**Comment by Subv** [comment]
You're missing the config file from the configuration savegame, extract it from your 3ds, place it in the right folder and try again

**Comment by fuhuan416** [comment]
But i load this game then the log window roll the core\memory.cpp not enough

**Comment by yuriks** [comment]
Please try #1097

**Comment by JohnGodgames** [comment]
fixed by #1097

**Comment by JohnGodgames** [comment]
#1097 was merged
